lib/shopify_api/resources/engagement.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyAPI
```

Comment:
Looks like the engagements resourse i.e. ContentEngagement is deprecated and we just write to a kafka stream instead